### ISSUE TITLE
代码运行的问题

### ISSUE THREAD

**Issue by ppt900** [renj-xu/NEGSC#1]
您好，在代码运行的过程中，我发现在In[8]中
le = LabelEncoder()
le.fit_transform(data.label.values)
data['label'] = 
label = data.label
data.drop(columns=['label'],inplace = True)
是否应该更正为
le = LabelEncoder()
data['label'] = le.fit_transform(data.label.values)
label = data.label
而且在剩余的模块中发现了大量的未定义变量
比如self.conv = GATConv(in_feat, out_feat, num_heads)中GATConv导入有问题
loss1 = b_xent(th.squeeze(logits), lbl)中b_xnet并没有定义
optimizer.zero_grad()中的optimizer也显示没有定义。
诸如此类的问题就不一一列举了。请问在开源代码的时候是不是错误的把旧版本当作最新版本公布了呢？这些问题希望可以考虑一下。

**Comment by ki511** [comment]
我用的也有未定义的内容，但是可以跑

**Comment by fhschina** [comment]
Same here. I’m not sure what you mean by running the code with unidentified variables, as that shouldn’t be possible. Could you please update the code so that it runs successfully?
